<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>메크로 테스터</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_comment">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>자동 댓글 시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_check">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>자동 출책 시작</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>36</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>-윤고딕340</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>1800.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>1800</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>39</y>
      <width>102</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>남은 출책 시간(초)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtLog">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>70</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
